app/main/controllers/template/rtmedia-actions.php

```diff
@@ -692,7 +692,7 @@ function rtm_album_media_count() {
 
 	if ( isset( $rtmedia_album_count_status ) && $rtmedia_album_count_status['status'] ) {
 		?>
-		<div class="rtmedia-album-media-count" title="<?php echo rtmedia_album_mediacounter() . RTMEDIA_MEDIA_LABEL; ?>">
+		<div class="rtmedia-album-media-count" title="<?php echo rtmedia_album_mediacounter() . ' ' . RTMEDIA_MEDIA_LABEL; ?>">
```

Comment:
Why you haven't used escaping for title attribute?

Reply:
@SID177 For maintaining consistency between photos, videos, and albums.

Reply:
Consistency for escaping?

Reply:
Done... I read sentence wrongly.